Main.c

```diff
@@ -0,0 +1,81 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// Test.c
+// Program on branch Test3.1
+// Program that implements TicTacToe.c.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <io.h>
+#include <fcntl.h>
+#include "TicTacToe.h"
+
+static wchar_t Get_Input (wchar_t* prompt) {

```

Comment:
No comment/description of the function.
Nested while loop is making the logic obscure.
Use of continue seems suspect.